ci-operator/jobs/openshift/microshift/openshift-microshift-main-presubmits.yaml

```diff
@@ -1,5 +1,77 @@
 presubmits:
   openshift/microshift:
+  - agent: kubernetes
+    always_run: false
```

Comment:
This means someone has to trigger it manually, right? I think that's what we want, until we have a better pass rate.

Reply:
I am aiming for the periodic jobs. I accidentally updated the wrong job.